index.html

```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                                </svg>
+
+                            </a>
+                        </li>
+                    </ul>
+
+                    <div class="mt-1">
+                        <a class="btn" href="#">Get my resume</a>
+                    </div>
+
+                </section>
+
+                <ul class="drop-buttons">
+                    <li class="drop-button">
+                        <p class="title">Languages</p>
+                        <ul class="languages extra-menu">
+                            <li>
+                                <img src="../images/javascript.png" alt="javascript icon" />
+                                <p>JavaScript</p>
```

Comment:
yeap, I forgot the existence of span, I was deciding between a title, a div, or p because my mind was more busy thinking about how to make the drop-menu functional without JavaScript.
I'm fixing it.